ios/Exponent/Versioned/Core/UniversalModules/EXNotifications/EXScopedNotificationsEmitter.m

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#if __has_include(<EXNotifications/EXNotificationsEmitter.h>)
+
+#import "EXScopedNotificationsEmitter.h"
+
+@interface EXScopedNotificationsEmitter ()
+
+@property (nonatomic, strong) NSString *experienceId;
+
+@end
+
+@implementation EXScopedNotificationsEmitter
+
+- (instancetype)initWithExperienceId:(NSString *)experienceId
+{
+  if (self = [super init]) {
+    _experienceId = experienceId;
+  }
+  
+  return self;
+}
+
+- (BOOL)_shouldHandleNotification:(UNNotification *)notification
+{
+  return [notification.request.content.userInfo[@"experienceId"] isEqual:_experienceId];
```

Comment:
Same here, if `experienceId` hasn't been provided we should YES handle the notification.